Dockerfile

```diff
@@ -181,13 +181,12 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # Build VMware Tools
-ENV OVT_VERSION 9.10.0
+ENV OVT_VERSION 10.0.0
 
-# use fixes for post 3.19 and 4.0 kernels from https://github.com/davispuh/open-vm-tools/tree/fixed and http://git.io/vIXpn
-# rebased onto the 9.10.0 release
-ENV OVT_REPO       https://github.com/cloudnativeapps/open-vm-tools
-ENV OVT_BRANCH     stable-9.10.x-kernel4-vmhgfs-fix
-ENV OVT_COMMIT     f654bdb390dd6753985379ea7df058aa8f6294ee
+# Pulling from the main repo on the stable 10.0.x release branch.
+ENV OVT_REPO       https://github.com/vmware/open-vm-tools
+ENV OVT_BRANCH     stable-10.0.x
+ENV OVT_COMMIT     85a99f4253dddf7e47e873fec4c43c15b7d8c2bc
```

Comment:
Oooh, since this repo has explicit releases, what are your thoughts on compiling from the release source tarballs instead of cloning? ie, something like:
```Dockerfile
ENV OVT_VERSION 10.0.0-3000743

RUN curl -fsSL "https://github.com/vmware/open-vm-tools/archive/open-vm-tools-${OVT_VERSION}.tar.gz" | tar -xz --strip-components=1 ...
```

(similar to how we handle the kernel up on line 26)